.github/workflows/ci-wrapper-rn.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     name: Publish unstable package release
     needs: build_test
     runs-on: macos-latest
-    if: "github.ref == 'refs/heads/master' && !contains(github.event.head_commit.message, 'chore(release): publish')"
+    if: "github.ref == 'refs/heads/master' && !(contains(github.event.head_commit.message, 'chore(release): publish') || contains(github.event.head_commit.message, 'chore(release): publish react-native'))"
```

Comment:
This is for the `unstable` releases, think we don't need to change here?

Reply:
Removed